.buildkite/pipeline.yml

```diff
@@ -18,10 +11,3 @@ steps:
 
   - label: ':python: Run Pyhton checks'
     command: tox -p auto -e "{py37,py38}-{test,doctest},pylint,mypy,report"
-
-  - wait
-
-  - label: "🚀Release"
```

Comment:
Why did you remove this? Now that you have access to Buildkite you can ensure that this isn't used in the PR pipeline so it should be okay now.